entries/5/500px.com.json

```diff
@@ -1,6 +1,9 @@
 {
   "500px": {
-    "domain": "web.500px.com",
+    "domain": "500px.com",
+    "additional-domains": [
+      "web.500px.com"
+    ],
```

Comment:
Since the parent domain is listed, the subdomain (www) isn't necessary.
```suggestion
```

Reply:
👌